scripts/loki_transform.py

```diff
@@ -489,7 +496,8 @@ def ecphys(mode, config, header, source, build, cpp, directive, frontend):
         block_dim = scheduler.config.dimensions['block_dim']
         transformation = SingleColumnCoalescedTransformation(
```

Comment:
Spotting the SCC wrapper here, I think the PR branch needs rebasing onto latest main